Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -4891,7 +4893,31 @@ public static void ListExecutedTriggers(Action<LootLockerListAllTriggersResponse
             }
             LootLockerAPIManager.ListAllExecutedTriggers(onComplete);
         }
+        #endregion
+
+        #region Triggers
 
+        /// <summary>
+        /// Invoke a set of triggers by key
+        ///
+        /// Note that the response contains two lists:
+        /// - One listing the keys of the triggers that were successfully executed
+        /// - One listing the triggers that failed as well as the reason they did so
+        ///
+        /// This means that the request can "succeed" but still contain triggers that failed. So make sure to check the inner results.
```

Comment:
One more thing that potentially could be added here:
Trigger X will only be called once, no matter how many times you send it, for example:
{triggerA, TriggerA, TriggerA, TriggerB}
Will call TriggerA once and TriggerB once.

Reply:
But also, this might be something that we make sure to add in Documentation instead, maybe not something to put in here. Fine by me either way!

Reply:
I think that goes to edge cases that should be covered in documentation.